src/node.rs

```diff
@@ -176,11 +178,122 @@ pub trait AudioControllableSourceNode {
     }
 }
 
+const TABLE_LENGTH_USIZE: usize = 2048;
+const TABLE_LENGTH_F32: f32 = TABLE_LENGTH_USIZE as f32;
+
+// Compute one period sine wavetable of size TABLE_LENGTH
+lazy_static! {
```

Comment:
Alternatively, in a build.rs script we can pre-compute these and later `include_bytes()` them. That saves an atomic sync and heap jump on every read. As always, profile before you optimize